examples/hosts/app-integration/external-data/src/view/taskListView.tsx

```diff
@@ -21,20 +21,21 @@ const TaskRow: React.FC<ITaskRowProps> = (props: ITaskRowProps) => {
     const { task, deleteTask } = props;
     const priorityRef = useRef<HTMLInputElement>(null);
     useEffect(() => {
-        const updateFromRemotePriority = () => {
+        const updateFromRemotePriority = (): void => {
             if (priorityRef.current !== null) {
                 priorityRef.current.value = task.priority.toString();
             }
         };
         task.on("priorityChanged", updateFromRemotePriority);
         updateFromRemotePriority();
-        return () => {
+        return (): void => {
             task.off("priorityChanged", updateFromRemotePriority);
         };
     }, [task]);
 
-    const inputHandler = (e) => {
-        const newValue = parseInt(e.target.value, 10);
+    const inputHandler = (e: React.FormEvent): void => {
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any, @typescript-eslint/no-unsafe-member-access
+        const newValue = Number.parseInt((e.target as any).value as string, 10);
```

Comment:
e.target should be an HTMLInputElement I think (unless it's actually some weird React thing, I can never keep that straight :-P) to eliminate the explicit `any` and `string` cast.

Reply:
The typing here was strange, and intellisense was failing to resolve it correctly for me. I'll give that type a shot.